Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -683,6 +683,34 @@ private class PrivacyPolicySetting: Setting {
     }
 }
 
+private class accountServiceSetting: WithoutAccountSetting {
+    override var accessoryType: UITableViewCellAccessoryType { return .None }
+    var prefs: Prefs { return settings.profile.prefs }
+    let prefKey = "accountService"
```

Comment:
This is too generic and needs to become something like `useChineseServices` I think.